### PR TITLE
create the header component

### PR DESCRIPTION
**Pull request by DanielAmah**
#### What does this PR do?
This PR creates the header component as stated in issue #789

### CODE REVIEW

**Comment by DanielAmah** [comment]
@nshki @baohouse @julianguyen. The PR is ready for review. Thanks.

**Comment by DanielAmah** [comment]
@julianguyen @nshki. I have updated the PR with the changes. Can you look and it and see if there are any other things I need to address thanks.

File: client/app/bundles/shared/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          </span>
+          <ul>
+            <UrlLink urllink="" secondTextholder="ABOUT" />
+            <UrlLink urllink="" secondTextholder="BLOG" />
+            <UrlLink urllink="" secondTextholder="RESOURCES" />
+            {
+              isLoggedIn
+                ? <UrlLink urllink="" firstTextHolder="DASHBOARD" secondTextholder= " / SIGN OUT" />
+                :
+                <UrlLink urllink="" secondTextholder="JOIN / SIGN IN" />
```
Comment (nshki) [inline client/app/bundles/shared/components/Header.jsx]: If you could make the text between the `/` for `UrlLink`s separate links, that would be great! Sorry if that wasn't clear!

File: client/app/bundles/shared/components/Header.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            font-size: $font14;
+            letter-spacing: 2px;
+          }
+        }
+      }
+    }
+
+    .dashboard {
+      color: $dashboard-link-color;
+    }
```
Comment (nshki) [inline client/app/bundles/shared/components/Header.scss]: Ahh I see what you're getting at here, and this is my fault for not making it clear in the designs, but "active" links (links that reflect what page you're currently on) should have this color, not specifically just the dashboard!

Comment (DanielAmah) [reply client/app/bundles/shared/components/Header.scss]: Thanks @nshki  for the feedback, I will get on it right away.

**Comment by nshki** [comment]
@DanielAmah I added a mock of the responsive version to the Figma document, by the way! You'll be able to see it to the left of the homepage mockup.

**Comment by DanielAmah** [comment]
I am looking for the mock. Can you pls drop the link again to the mock of the responsive version. Thanks @nshki

**Comment by nshki** [comment]
@DanielAmah Here's the link! https://www.figma.com/file/RTjPkO0nSuJcMqqWLjeVdEof/if-me?node-id=65%3A618

Once you log into Figma, you should be able to switch what screen you're looking at by clicking on the element right beneath the menu icon in the top left (it should say <Screen Name> <Num> of 3). From there, go to "Screens" and you'll see the responsive header in the top left.

**Comment by julianguyen** [comment]
Can you rebase from master? :)

**Comment by Seunadex** [comment]
@nshki @julianguyen I'm going to finish up this PR

**Pull request closed without merging**